src/bin/common/parse_command.ml

```diff
@@ -0,0 +1,1157 @@
+    Arg.(value & opt int 0 & info ["gc-policy"] ~docv ~docs ~doc) in
+
+  Term.(ret (const mk_internal_opt $
+             disable_weaks $ enable_assertions $ gc_policy
+            ))
+
+let parse_limit_opt =
+
+  let docs = s_limit in
+
+  let age_bound =
+    let doc = "Set the age limit bound." in
+    let docv = "AGE" in
+    Arg.(value & opt int (get_age_bound ()) &
+         info ["age-bound"] ~docv ~docs ~doc) in
+
+  let fm_cross_limit =
+    (* TODO : Link this to Alt-Ergo numbers *)
+    let dv = Numbers.Q.to_string (get_fm_cross_limit ()) in
+    let doc = Format.sprintf
+        "Skip Fourier-Motzkin variables elimination steps that may produce \
+         a number of inequalities that is greater than the given limit. \
+         However, unit eliminations are always done." in
+    let docv = "VAL" in
+    Arg.(value & opt string dv & info ["fm-cross-limit"] ~docv ~docs ~doc) in
+
+  let timelimit_interpretation =
+    let doc = "Set the time limit to $(docv) seconds for model generation \
+               (not supported on Windows)." in
+    let docv = "SEC" in
+    Arg.(value & opt (some float) None &
+         info ["timelimit-interpretation"] ~docv ~docs ~doc) in
+
+  let steps_bound =
+    let doc = "Set the maximum number of steps." in
+    let docv = "STEPS" in
+    Arg.(value & opt int (get_steps_bound ()) &
+         info ["S"; "steps-bound"] ~docv ~doc) in
+
+  let timelimit =
+    let doc =
+      "Set the time limit to $(docv) seconds (not supported on Windows)." in
+    let docv = "VAL" in
+    Arg.(value & opt (some float) None & info ["t"; "timelimit"] ~docv ~doc) in
+
+  let timelimit_per_goal =
+    let doc =
+      "Set the given timelimit for each goal, in case of multiple goals per \
+       file. In this case, time spent in preprocessing is separated from \
+       resolution time. Not relevant for GUI-mode." in
+    Arg.(value & flag & info ["timelimit-per-goal"] ~docs ~doc) in
+
+  Term.(ret (const mk_limit_opt $
+             age_bound $ fm_cross_limit $ timelimit_interpretation $
+             steps_bound $ timelimit $ timelimit_per_goal
+            ))
+
+let parse_output_opt =
+
+  let docs = s_output in
+
+  let interpretation =
+    let doc =
+      "Experimental support for counter-example generation. Possible \
+       values are 1, 2, or 3 to compute an interpretation before returning \
+       Unknown, before instantiation, or before every decision or \
+       instantiation. A negative value (-1, -2, or -3) will disable \
+       interpretation display. Note that $(b, --max-split) limitation will \
+       be ignored in model generation phase." in
+    let docv = "VAL" in
+    Arg.(value & opt int (get_interpretation ()) &
+         info ["interpretation"] ~docv ~docs ~doc) in
+
+  let model =
+    let doc = Format.sprintf
+        "Experimental support for models on labeled terms. \
+         $(docv) must be %s. %s shows a complete model and %s shows \
+         all models."
```

Comment:
Is the difference only in the printing/showing of the model or also in the generation ? The documentation should be precise about this, ^^

Reply:
I don't know, do you?